### DISCUSSION TITLE
Simulation of particle polarization

### DISCUSSION THREAD

**Discussion by eric11210** [nrc-cnrc/EGSnrc#973] [Physics] [answered]
Should the polarization of photons (or indeed all particles) be included in the simulation? Some of the cross sections do have a polarization dependence, and bremsstrahlung produces photons with some degree of polarity, depending on electron and photon energy, and photon angle (see for example Gluckstern and Hull, "Polarization Dependence of the Integrated Bremsstrahlung Cross Section," Physical Review 90(6), 1953).

**Comment by ftessier** [comment]
Currently EGSnrc does _not_ handle polarization. I believe @mainegra can provide more insight in this regard.

**Reply by eric11210** [reply]
Thanks for confirming, I didn't think so. Do you or others think it worth pursuing? Looking at the documentation, it seems that Geant4 for example has the option of including the effects of polarization. Do you know of any studies looking at the impact of polarization in radiation transport?

**Comment by mchamberland** [comment]
A very cursory glance at MCNP and Penelope documentation does not yield any reference to polarization, except in the case of polarization of the medium when talking about density correction effects.

As @eric11210 points out, polarization is implemented in Geant4, though I wonder if it's more targeted towards optical photon simulations?

**Reply by eric11210** [reply]
Thanks for looking into MCNP and Penelope as well.

According to the docs, Geant4 includes polarization effects for Bhabha/Møller scattering, positron annihilation, Compton scattering, pair creation, and bremsstrahlung.

**Reply by ftessier** [reply]
[From our own docs](https://github.com/nrc-cnrc/EGSnrc/blob/3cc786bb6e1ce9948e3e63285eeb09ed7c2f71b4/HEN_HOUSE/doc/src/pirs701-egsnrc/inputs/preface.tex#L160-L164), it appears that `EGS5` has implemented photon polarization.

**Reply by eric11210** [reply]
I did notice that.

**Reply by ftessier** [reply]
Looking into [EGS5 manual](https://rcwww.kek.jp/research/egs/egs5_manual/slac730-150228.pdf), there are references to an `EGS4` implementation via `ausgab` in the 1990s:

> Recent paper by Flöttmann [59] and others[93, 98] introduced polarization into EGS4, by means of the additionalcalls-to-AUSGAB feature, in order to investigate the development of high-intensity positron sources for future linear colliders.

**Reply by ftessier** [reply]
According to the [Geant4 manual:](https://geant4-userdoc.web.cern.ch/UsersGuides/PhysicsReferenceManual/html/electromagnetic/polarized_incident/polarisation.html)

> (...) the most complete has been developed by K. Flöttmann [[Flottmann93a]](https://geant4-userdoc.web.cern.ch/UsersGuides/PhysicsReferenceManual/html/electromagnetic/polarized_incident/polarisation.html#em-floettmannthesis). It is based on the matrix formalism [[McM61]](https://geant4-userdoc.web.cern.ch/UsersGuides/PhysicsReferenceManual/html/electromagnetic/polarized_incident/polarisation.html#em-mcmaster1961), which enables a very general treatment of polarization.

**Reply by ojalaj** [reply]
@mchamberland My understanding (and very small experience) is that polarization effect has its role, or is even a pre-requisite for accurate synchrotron based microbeam radiotherapy simulations.

**Reply by mchamberland** [reply]
@ojalaj today I learned something! 😄

**Reply by ojalaj** [reply]
Yeah, I got to know this years ago, when I was asked to perform such simulations and discussed with collaborators about the physics requirements for the simulations.

**Comment by ftessier** [comment, marked as answer]
I added polarization as a research project idea in #790.

**Comment by mainegra** [comment]
The KEK group (Namito and Hirayama) out of Tsukuba, Japan, had implemented linear polarization of photon beams back in 1995. They point out that it is important for modeling synchrotron beams as @ojalaj mentioned above, but also for PET imaging simulations. Here is a quote from [their 1993 paper](https://doi.org/10.1016/0168-9002(93)90770-I) on their EGS4 implementation: 

> linearly polarized photon scattering is essentially important in simulating accidental coincidence background in PET (positron emission tomography) [14] . This is because the two annihilation photons are polarized perpendicular to each other.

They have a [website dedicated to their low-energy improvements to EGS4](http://rcwww.kek.jp/research/egs/kek/egs4/lowen.html), which have probably been implemented in EGS5.

**Comment by eric11210** [comment]
Thank you everyone for the discussion! It looks like polarization is indeed important in certain applications, and would therefore be a worthwhile project to pursue.
